torch/testing/_internal/common_quantized.py

```diff
@@ -479,3 +479,110 @@ def to_blocked(input_matrix) -> torch.Tensor:
     rearranged = blocks.reshape(-1, 4, 32, 4).transpose(1, 2).reshape(-1, 32, 16)
 
     return rearranged.flatten()
+
+# This function is extracted from https://github.com/pytorch/ao/blob/v0.12.0/torchao/prototype/mx_formats/mx_tensor.py#L142
+def to_mxfp8(
```

Comment:
we already have this logic here (https://github.com/pytorch/pytorch/blob/081cab045472ce045634548cc6c14a4870641e23/test/test_matmul_cuda.py#L931), want to update that with RCEIL instead? Can be in a future PR.